charts/sourcegraph/values.yaml

```diff
@@ -322,6 +322,52 @@ embeddings:
   extraVolumeMounts: {}
   extraVolumes: {}
 
+qdrant:
+  # -- Enable `qdrant`
+  enabled: false
+  # -- Name of the `qdrant` service
+  name: qdrant
+  image:
+    # -- Docker image name for the `embeddings` image
+    name: "qdrant"
+    # -- Docker image tag for the `embeddings` image
+    defaultTag: "239247_2023-08-18_5.1-433e1b1c997f@sha256:eafcd7af2aca699fa9c9ce8e6aa674cc0470441f794baf031296d5d1cdadd0bc"
+  # -- Resource requests & limits for the `qdrant` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)
+  resources:
+    limits:
+      cpu: "2"
+      memory: 8G
+    requests:
+      cpu: "500m"
+      memory: 2G
```

Comment:
Are there any guidelines for sizing on cloud, other than just "whatever is needed for perf"?

Reply:
in cloud, we will provide our own override.

- https://github.com/sourcegraph/cloud/blob/main/overrides/global.override.yaml
- https://github.com/sourcegraph/cloud/blob/main/overrides/small-resources.override.yaml

generally start small and increase as needed

Reply:
got it, thanks!